frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -164,7 +164,7 @@ const toApiProperty = (property: IProperty): IApiProperty => {
     parcelId: property.propertyTypeId === 0 ? property.id : undefined,
     buildingId: property.propertyTypeId === 1 ? property.id : undefined,
     pid: property.pid,
-    pin: property.pin,
+    pin: Number(property.pin),
```

Comment:
Was causing the bad request on project creation, was doing this in test too. I guess we just never really submitted properties with PINs that often